src/NzbDrone.Core/Download/CompletedDownloadService.cs

```diff
@@ -94,15 +94,15 @@ public void Check(TrackedDownload trackedDownload)
 
                 if (series == null)
                 {
-                    trackedDownload.Warn("Series title mismatch; automatic import is not possible.");
+                    trackedDownload.Warn("Series title mismatch; automatic import is not possible. See the related entry on the download troubleshooting article for details.");
```

Comment:
This seconds sentence seems a little wordy. Maybe just `Check the download troubleshooting entry on the wiki for common causes.`?